examples/basics/src/components/Card.astro

```diff
@@ -14,7 +14,6 @@ const {href, title, body} = Astro.props;
         </h2>
         <p>
             {body}
-            <slot name="icon" />
```

Comment:
Just making a note here that the action being taken is the removal of the `<slot name="icon">`
this would unblock the build from hanging